db/revision_cache_bypass.go

```diff
@@ -0,0 +1,55 @@
+package db
+
+import "github.com/pkg/errors"
+
+// BypassRevisionCache is an implementation of the RevisionCache interface that does perform any caching.
+// For any Get operation, it will always immediately fetch the requested revision from the bucket.
+type BypassRevisionCache struct {
+	loaderFunc RevisionCacheLoaderFunc
+}
+
+// Get fetches the revision for the given docID and revID immediately from the bucket.
+func (rc *BypassRevisionCache) Get(docID, revID string, copyType BodyCopyType) (docRev DocumentRevision, err error) {
+	if rc.loaderFunc == nil {
+		return DocumentRevision{}, errors.New("nil loaderFunc")
+	}
```

Comment:
Would it be better to have a constructor that validates that a loaderFunc is provided when using BypassRevisionCache, to avoid having to validate on each Get?